string area1
string area2
string area3
string area4